tools/mkwinsyscall/mkwinsyscall.go

```diff
@@ -580,26 +573,26 @@ func (f *Fn) HelperCallParamList() string {
 }
 
 // MaybeAbsent returns source code for handling functions that are possibly unavailable.
-func (p *Fn) MaybeAbsent() string {
-	if !p.Rets.fnMaybeAbsent {
+func (f *Fn) MaybeAbsent() string {
+	if !f.Rets.fnMaybeAbsent {
 		return ""
 	}
 	const code = `%[1]s = proc%[2]s.Find()
 	if %[1]s != nil {
 		return
 	}`
-	errorVar := p.Rets.ErrorVarName()
+	errorVar := f.Rets.ErrorVarName()
 	if errorVar == "" {
 		errorVar = "err"
 	}
-	return fmt.Sprintf(code, errorVar, p.DLLFuncName())
+	return fmt.Sprintf(code, errorVar, f.DLLFuncName())
 }
 
 // IsUTF16 is true, if f is W (utf16) function. It is false
 // for all A (ascii) functions.
```

Comment:
Should update the comment to talk about the second condition this now checks (utf16 flag)